dao/src/main/java/org/thingsboard/server/dao/device/DeviceConnectivityServiceImpl.java

```diff
@@ -197,6 +201,10 @@ private Resource getCert(String path) throws Exception {
                     pemContentBuilder.append("-----END CERTIFICATE-----\n");
                 }
             }
+        } catch (IOException e) {
```

Comment:
Hi @dashevchenko, I noticed that this catch clause was moved from the getPemCertFile method. The original catch clause caught the Exception class, while you are now only catching IOException. Are you sure that no other exceptions extending Exception might be thrown?